### PR TITLE
[Docs] Make mlrun-kit the main installation option

### PR DESCRIPTION
**Pull request by gilad-shaham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +91,7 @@ pip install mlrun
 MLRun requires separate containers for the API and the dashboard (UI).
 You can also select to use the pre-baked JupyterLab image.
 
-To install and run MLRun locally using Docker or Kubernetes, see the instructions in [**hack/local/README.md**](hack/local/README.md).
+To install and run MLRun locally using Docker or Kubernetes, see the instructions in the [**MLRun documentation**]([hack/local/README.md](https://mlrun.readthedocs.io/en/latest/install.html)).
```
Comment (Hedingber) [inline README.md:94]: The new link doesn't work, looks like a typo

File: README.md [hunk 2]
```diff
@@ -126,21 +145,33 @@ MLRun has many code examples and tutorial Jupyter notebooks with embedded docume
...
-  - [The MLRun Dashboard](#mlrun-ui)
-  - [MLRun Database Methods](#mlrun-db-methods)
-- [Additional Information and Examples](#additional-info-n-examples)
-  - [Replacing Runtime Context Parameters from the CLI](#replace-runtime-context-param-from-cli)
-  - [Remote Execution](#remote-execution)
-- [Running an MLRun Service](#run-mlrun-service)
-  - [Using the MLRun CLI](#run-mlrun-service-cli)
+- [Benefits](#benefits)
+- [Components](#components)
+    - [In This Document](#in-this-document)
```
Comment (Hedingber) [inline README.md:150]: In this document seems unrelated as a sub-title of components suggest to return it to the top as before plus changing it from `####` to `##`

File: docs/install.md [hunk 3]
```diff
@@ -2,119 +2,224 @@
...
 
-<a id=k8s-create-a-namespace></a>
-### Create a namespace
+- `<your-registry-server>` is your Private Docker Registry FQDN. (https://index.docker.io/v1/ for Docker Hub).
+- `<your-username>` is your Docker username.
+- `<your-password>` is your Docker password.
+- `<your-email>` is your Docker email.
 
-Create a namespace for mlrun. For example:
+To install the chart with the release name `my-mlrun` use the following command.
```
Comment (Hedingber) [inline docs/install.md:71]: I would have rename `my-mlrun` to `mlrun-kit` 
no functional change just feels more professional IMO
this is just a suggestion, you don't have to take it

File: docs/install.md [hunk 4]
```diff
@@ -2,119 +2,224 @@
...
+1. Make sure the MLRun version installed with the MLRun Kit is the same as the MLRun version on your remote cluster. If needed, upgrade MLRun either in your local installation or on the remote cluster so they would match.
+2. Ensure that you have remote access to your MLRun service (i.e., to the service URL on the remote Kubernetes cluster).
 
-1. Run the following commands (provided Helm is installed):
-    ```sh
-    helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-    helm install -n mlrun nfsprov stable/nfs-server-provisioner
+#### Set Environment Variables
+
+Define your MLRun configuration. As a minimum requirement &mdash;
```
Comment (Hedingber) [inline docs/install.md:123]: the `&mdash;` is on purpose ?

**Review by Hedingber — CHANGES_REQUESTED** [review]
Looks good
several minor comments

File: README.md (hunk 1 above)
Comment (gilad-shaham) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (gilad-shaham) [reply README.md]: fixed

File: docs/install.md (hunk 3 above)
Comment (gilad-shaham) [reply docs/install.md]: done

File: docs/install.md (hunk 4 above)
Comment (gilad-shaham) [reply docs/install.md]: fixed (also in remote.md)

**Review by Hedingber — APPROVED** [review]
